code-ext/chrome/background.js

```diff
@@ -26,7 +26,31 @@ chrome.tabs.onUpdated.addListener(async (tabId, changeInfo, tab) => {
     } catch (ex) {
       console.error(ex);
     }
-  }
+  
+});
+
+
+chrome.tabs.onUpdated.addListener(async (tabId, changeInfo, tab) => {
```

Comment:
Why are these commented out sa chrome script?

Reply:
change requirements according to tasks  tabs.onUpdated is injection on tab load or entering webpage sa tab injection natin is via icon click nmn db ??

Reply:
So once we clicked the button disable which is done in React, the chrome script can communicate in it?

Reply:
So once we clicked the button disable which is done in React, the chrome script can communicate in it?

Reply:
so far wla pa ko nakita na pang tanggal ng injection ang thinking ko would be to reinject the code everytime i click ung icon

Reply:
although wla ko nag expect na mag remain ang state that is why nag butang ko comment na need pa i refresh ang page b4 mag reinject ulit

Reply:
Alright. Okay I think this looks good.